[2025-04-11T07:50:52.929+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-11T07:50:52.935+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_dag.say_hello manual__2025-04-11T07:50:52.479642+00:00 [queued]>
[2025-04-11T07:50:52.939+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_dag.say_hello manual__2025-04-11T07:50:52.479642+00:00 [queued]>
[2025-04-11T07:50:52.939+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-11T07:50:52.944+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): say_hello> on 2025-04-11 07:50:52.479642+00:00
[2025-04-11T07:50:52.949+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'hello_dag', 'say_hello', 'manual__2025-04-11T07:50:52.479642+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpbf7chvap']
[2025-04-11T07:50:52.951+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=404) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-11T07:50:52.951+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask say_hello
[2025-04-11T07:50:52.951+0000] {standard_task_runner.py:72} INFO - Started process 405 to run task
[2025-04-11T07:50:52.972+0000] {task_command.py:467} INFO - Running <TaskInstance: hello_dag.say_hello manual__2025-04-11T07:50:52.479642+00:00 [running]> on host b9b73229c5e1
[2025-04-11T07:50:53.011+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_dag' AIRFLOW_CTX_TASK_ID='say_hello' AIRFLOW_CTX_EXECUTION_DATE='2025-04-11T07:50:52.479642+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-11T07:50:52.479642+00:00'
[2025-04-11T07:50:53.012+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-11T07:50:53.027+0000] {logging_mixin.py:190} INFO - Hello depuis mon premier DAG !
[2025-04-11T07:50:53.028+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-11T07:50:53.035+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-11T07:50:53.036+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=hello_dag, task_id=say_hello, run_id=manual__2025-04-11T07:50:52.479642+00:00, execution_date=20250411T075052, start_date=20250411T075052, end_date=20250411T075053
[2025-04-11T07:50:53.087+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-11T07:50:53.093+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-11T07:50:53.094+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
